README.md

```diff
@@ -262,6 +262,21 @@ String result = metadata.get("Result");
 
 Mainly for debug purposes, jchess provides a simple API to print a position in CLI:
 
+#### Unicode
+
+The unicode renderer's output is compact but does not support showing which cells are dark or light when they are not empty.
+
+```java
+JChessGame game = JChessGame.newGame();
+UnicodePositionRenderer.render(System.out, game.getPosition());
+```
+
+![image](https://user-images.githubusercontent.com/18663098/159409079-d4f994d3-2051-404a-aec7-951e0a4a7755.png)
```

Comment:
The picture doesn't show the initial position. Can you please also commit the image in the repo to avoid link to third-party site.